solr/solrj/src/java/org/apache/solr/client/solrj/impl/LBHttp2SolrClient.java

```diff
@@ -301,14 +301,33 @@ public void onFailure(Throwable oe) {
   public static class Builder {
     private final Http2SolrClient http2Client;
     private final String[] baseSolrUrls;
+    private int interval;
```

Comment:
aliveCheckInterval would be clearer!

Reply:
On the other hand, this merely uses the very same field name used in the client so... let's either change them all or leave them all (as you wish).